src/Microsoft.Azure.SignalR.Common/ServiceConnections/scopes/ClientConnectionScope.cs

```diff
@@ -30,7 +30,7 @@ protected internal ClientConnectionScope(IServiceConnection outboundConnection,
                                 Properties = new ClientConnectionScopeProperties()
                                 {
                                     OutboundServiceConnection = outboundConnection,
-                                    IsDiagnosticClient = isDiagnosticClient
+                                    IsDiagnosticClient = isDiagnosticClient,
```

Comment:
>IsDiagnosticClient [](start = 36, length = 18)

where will this property be used? Is this also a relevant property to this PR?

Reply:
reverted, accidently added a `,`